certdepot/mongo_depot_test.go

```diff
@@ -0,0 +1,382 @@
+package certdepot
+
+import (
+	"testing"
+	"time"
+
+	"github.com/square/certstrap/depot"
+	"github.com/stretchr/testify/suite"
+	mgo "gopkg.in/mgo.v2"
+)
+
+type MongoDepotTestSuite struct {
```

Comment:
I just thought it might be cool to add to your test running against the file depot as a way of validating the behavior of your depot implementation.

Reply:
I agree, I will create a subsequent ticket for that.